### PR TITLE
Add automation to Leprechaun Magic

### PR DESCRIPTION
**Pull request by TikaelSol**

### CODE REVIEW

File: packs/pfs-season-4-bestiary/house-spirits-with-a-large-number-of-regenerating-pies.json [hunk 1]
```diff
@@ -0,0 +1,182 @@
...
+            "isComplex": true,
+            "level": {
+                "value": 3
+            },
+            "reset": "",
+            "routine": "<p>(3 actions) The hazard regenerates any pies that it has thrown at a cost of one action per pie (to a maximum of 3 active pies). Each successful check to disable the hazard deactivates one of the pies, reducing the number of active pies by 1. After regenerating pies on its turn, the hazard throws pies whenever a creature triggers continually sur-piesed. For each pie Strike, roll [[/r 1d4]]. On a 1, the pie pan has nails driven into it by Numblekin and deals and additional [[/r (1d10+6)[piercing]]] damage. The nails make the pie's damage lethal, rather than nonlethal.</p>",
```
Comment (simonward) [inline packs/pfs-season-4-bestiary/house-spirits-with-a-large-number-of-regenerating-pies.json:149]: “… deals an additional…”

File: packs/pfs-season-4-bestiary/house-spirits-with-an-absurd-number-of-regenerating-pies-copy.json [hunk 2]
```diff
@@ -0,0 +1,182 @@
...
+                    ]
+                },
+                "weaponType": {
+                    "value": "ranged"
+                }
+            },
+            "type": "melee"
+        }
+    ],
+    "name": "House Spirits With an Absurd Number of Regenerating Pies (Copy)",
```
Comment (simonward) [inline packs/pfs-season-4-bestiary/house-spirits-with-an-absurd-number-of-regenerating-pies-copy.json:121]:
```suggestion
    "name": "House Spirits With an Absurd Number of Regenerating Pies",
```

File: packs/pfs-season-4-bestiary/house-spirits-with-an-absurd-number-of-regenerating-pies.json [hunk 3]
```diff
@@ -118,7 +118,10 @@
             "type": "melee"
         }
     ],
-    "name": "House Spirits With an Absurd Number of Regenerating Pies (Copy)",
+    "name": "House Spirits With an Absurd Number of Regenerating Pies",
+    "prototypeToken": {
+        "name": "House Spirits With an Absurd Number of Regenerating Pies (Copy)"
```
Comment (simonward) [inline packs/pfs-season-4-bestiary/house-spirits-with-an-absurd-number-of-regenerating-pies.json:123]: Is this correct?

File: packs/pfs-season-4-bestiary/house-spirits-with-an-absurd-number-of-regenerating-pies.json [hunk 4]
```diff
@@ -0,0 +1,185 @@
...
+                }
+            },
+            "type": "melee"
+        }
+    ],
+    "name": "House Spirits With an Absurd Number of Regenerating Pies",
+    "prototypeToken": {
+        "name": "House Spirits With an Absurd Number of Regenerating Pies (Copy)"
+    },
+    "system": {
```
Comment (TikaelSol) [inline packs/pfs-season-4-bestiary/house-spirits-with-an-absurd-number-of-regenerating-pies.json:125]:
```suggestion
    "name": "House Spirits With an Absurd Number of Regenerating Pies",
    "system": {
```

**Pull request merged**